subsys/bluetooth/controller/hci/hci.c

```diff
@@ -1688,6 +1688,50 @@ static void le_read_tx_power(struct net_buf *buf, struct net_buf **evt)
 	ll_tx_pwr_get(&rp->min_tx_power, &rp->max_tx_power);
 }
 
+#if IS_ENABLED(CONFIG_BT_CTLR_DF)
+#if IS_ENABLED(CONFIG_BT_CTLR_DF_CONN_CTE_RSP)
+static void le_df_set_conn_cte_tx_params(struct net_buf *buf,
+					 struct net_buf **evt)
+{
+	struct bt_hci_cp_le_set_conn_cte_tx_params *cmd = (void *)buf->data;
+	struct bt_hci_rp_le_set_conn_cte_tx_params *rp;
+	uint16_t handle = sys_le16_to_cpu(cmd->handle);
+	uint8_t status;
+
+	status = ll_df_set_conn_cte_tx_params(handle, cmd->cte_types,
+					      cmd->switch_pattern_len,
+					      cmd->ant_id);
+
+
+
+	rp = hci_cmd_complete(evt, sizeof(*rp));
+
+	rp->status = status;
+	rp->handle = sys_cpu_to_le16(handle);
+}
+#endif /* CONFIG_BT_CTLR_DF_CONN_CTE_RSP */
+
+static void le_df_read_ant_inf(struct net_buf *buf, struct net_buf **evt)
+{
+	struct bt_hci_rp_le_read_ant_info *rp;
+	uint8_t switch_sample_rates;
+	uint8_t num_ant;
+	uint8_t max_switch_pattern_len;
+	uint8_t max_cte_len;
```

Comment:
```suggestion
	struct bt_hci_rp_le_read_ant_info *rp;
	uint8_t max_switch_pattern_len;
	uint8_t switch_sample_rates;
	uint8_t max_cte_len;
	uint8_t num_ant;
```

Reverse Christmas tree order

Reply:
changed to suggested order